overlord/snapstate/autorefresh.go

```diff
@@ -58,6 +58,28 @@ var (
 // refreshRetryDelay specified the minimum time to retry failed refreshes
 var refreshRetryDelay = 20 * time.Minute
 
+// refreshCandidate carries information about a single snap to update as part
+// of auto-refresh.
+type refreshCandidate struct {
+	SnapSetup
+}
+
+func (rh *refreshCandidate) Type() snap.Type {
```

Comment:
(nitpick^2) why the name `rh` for refreshCandidate? mostly curious, does not really matter

Reply:
Ah, because it was named "refreshHint" initially (hence "rh"), then renamed to refreshCandidate. Will rename this, thanks!